src/greedy.jl

```diff
@@ -13,6 +19,7 @@ before. `shift` parameter is used for LPRC.
 function update_late_violation!(solution::Solution, nb::Int, last::Int,
                                 p::Array{Int, 1}, q::Array{Int, 1},
                                 flag::Array{Bool, 2}, shift::Int = 0)
+    @warn "Function update_late_violation! depreciated -> call functions in solution.jl"
```

Comment:
Chose deprecated or obsolete. Depreciated means that the monetary value has dropped